packages/vite/package.json

```diff
@@ -20,20 +20,14 @@
   "types": "./dist/node/index.d.ts",
   "exports": {
     ".": {
-      "import": {
-        "types": "./dist/node/index.d.ts",
-        "default": "./dist/node/index.js"
-      },
-      "require": {
-        "types": "./index.d.cts",
-        "default": "./index.cjs"
-      }
+      "module-sync": "./dist/node/index.js",
+      "import": "./dist/node/index.js",
+      "require": "./index.cjs"
     },
     "./client": {
       "types": "./client.d.ts"
     },
     "./module-runner": {
-      "types": "./dist/node/module-runner.d.ts",
       "import": "./dist/node/module-runner.js"
     },
```

Comment:
Maybe we could simplify this as `"./module-runner": "./dist/node/module-runner.js"`. Usually `import` condition is only useful if there's a respective `require` condition.

Reply:
Updated 👍